src/frontends/tensorflow/src/translate_session.cpp

```diff
@@ -446,7 +446,8 @@ void TranslateSession::translate_graph(const ov::frontend::InputModel::Ptr& inpu
         }
 
         auto param = std::make_shared<ov::op::v0::Parameter>(input_type, input_shape);
-        set_node_name(input_name, param);
+        param->set_friendly_name(input_name);
+        set_out_name(input_name, param);
```

Comment:
Can we have unit-test to cover the following cases:
1) Have a simple model we pass to input option "param1,param2" node names (not tensor). It will check that tensor names along with nodes names will have single names "param1,param2". Let us add it into TF FE unit-tests with checking tensor names
2) Have a simple model we pass to input option "param1:0,param2:0" node names (not tensor). It will check that tensor names along with nodes names will have single names "param1:0,param2:0". Let us add it into TF FE unit-tests with checking tensor names
3) Have a simple model when Placeholder is going into Identity and we will check that it contains single tensor name

Reply:
Added tests, but current behavior is following. If we set node names "param1", "param2" we still have tensor names "param1:0" and "param2:0".

`input_name` here comes from InputModel input names:
https://github.com/openvinotoolkit/openvino/blob/7423fc52a42a441509727b06c0907143a3481a51/src/frontends/tensorflow/src/translate_session.cpp#L449

For TF1 models InputModel always creates ":0" input so for TF1 we always have ":0" input name, though we can apply preprocessing by operation names, as we have aliases here: 
https://github.com/openvinotoolkit/openvino/blob/7423fc52a42a441509727b06c0907143a3481a51/src/frontends/tensorflow/src/input_model.cpp#L231

Operation name in [this line](https://github.com/openvinotoolkit/openvino/blob/7423fc52a42a441509727b06c0907143a3481a51/src/frontends/tensorflow/src/translate_session.cpp#L449) appears only for TF2 models in subgraphs.

Reply:
Added logic to set op names in case if user specified them and changed new tests to check op name. Please check.

Reply:
Implemented the logic as discussed, please check.